### ISSUE TITLE
[ci] build break of sonic-telemetry

### ISSUE THREAD

**Issue by qiluo-msft** [Azure/sonic-buildimage#6815]
#### Description

When we are working on a mellanox related PR https://github.com/Azure/sonic-buildimage/pull/6805 which targeting 201911 branch, we met a broadcom PR build failure https://dev.azure.com/mssonic/build/_build/results?buildId=4220&view=logs&j=406f67d9-8952-5074-7a53-dcb346cb6a5f&t=b3777219-5c97-5c20-138a-8b0d961575dd

I suspect it is the relating to build sequence in parallel build.

```
2021-02-18T02:36:33.3344475Z [ building ] [ target/debs/stretch/sonic-mgmt-framework_1.0-01_amd64.deb ] 
2021-02-18T02:36:37.4617616Z [ finished ] [ target/debs/stretch/sonic-telemetry_0.1_amd64.deb ] 
2021-02-18T02:36:37.4618794Z [ FAIL LOG START ] [ target/debs/stretch/sonic-telemetry_0.1_amd64.deb ]
2021-02-18T02:36:37.4801262Z /sonic/src/sonic-telemetry /sonic
2021-02-18T02:36:37.4802434Z dpkg-buildpackage: info: source package sonic-telemetry
2021-02-18T02:36:37.4803399Z dpkg-buildpackage: info: source version 0.1
2021-02-18T02:36:37.4804355Z dpkg-buildpackage: info: source distribution UNRELEASED
2021-02-18T02:36:37.4805430Z dpkg-buildpackage: info: source changed by Jipan Yang <jipan.yang@alibaba-inc.com>
2021-02-18T02:36:37.4806402Z  dpkg-source --before-build sonic-telemetry
2021-02-18T02:36:37.4807313Z dpkg-buildpackage: info: host architecture amd64
2021-02-18T02:36:37.4807867Z  fakeroot debian/rules clean
2021-02-18T02:36:37.4810278Z make[1]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
2021-02-18T02:36:37.4811543Z make[1]: Entering directory '/sonic/src/sonic-telemetry'
2021-02-18T02:36:37.4812147Z dh clean 
2021-02-18T02:36:37.4812565Z    dh_auto_clean
2021-02-18T02:36:37.4813704Z 	make -j1 clean
2021-02-18T02:36:37.4814921Z make[2]: Entering directory '/sonic/src/sonic-telemetry'
2021-02-18T02:36:37.4815902Z rm -rf cvl
2021-02-18T02:36:37.4816903Z rm -rf translib
2021-02-18T02:36:37.4817839Z rm -rf vendor
2021-02-18T02:36:37.4818604Z chmod -f -R u+w /tmp/go/pkg || true
2021-02-18T02:36:37.4819331Z rm -rf /tmp/go
2021-02-18T02:36:37.4819986Z rm -f src
2021-02-18T02:36:37.4820801Z make[2]: Leaving directory '/sonic/src/sonic-telemetry'
2021-02-18T02:36:37.4821488Z    dh_clean
2021-02-18T02:36:37.4822277Z make[1]: Leaving directory '/sonic/src/sonic-telemetry'
2021-02-18T02:36:37.4822781Z  debian/rules build
2021-02-18T02:36:37.4823744Z make[1]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
2021-02-18T02:36:37.4824777Z make[1]: Entering directory '/sonic/src/sonic-telemetry'
2021-02-18T02:36:37.4825273Z dh build 
2021-02-18T02:36:37.4825658Z    dh_update_autotools_config
2021-02-18T02:36:37.4826087Z    dh_auto_configure
2021-02-18T02:36:37.4826464Z    dh_auto_build
2021-02-18T02:36:37.4827319Z 	make -j1
2021-02-18T02:36:37.4828263Z make[2]: Entering directory '/sonic/src/sonic-telemetry'
2021-02-18T02:36:37.4828959Z rm -rf cvl
2021-02-18T02:36:37.4829554Z rm -rf translib
2021-02-18T02:36:37.4830664Z cp -r ../sonic-mgmt-framework/src/cvl ./
2021-02-18T02:36:37.4831459Z cp -r ../sonic-mgmt-framework/src/translib ./
2021-02-18T02:36:37.4832497Z find cvl -name \*\.go -exec sed -i -e 's/\"translib/\"github.com\/Azure\/sonic-telemetry\/translib/g' {} \;
2021-02-18T02:36:37.4833778Z find translib -name \*\.go -exec sed -i -e 's/\"translib/\"github.com\/Azure\/sonic-telemetry\/translib/g' {} \;
2021-02-18T02:36:37.4835009Z find cvl -name \*\.go -exec sed -i -e 's/\"cvl/\"github.com\/Azure\/sonic-telemetry\/cvl/g' {} \;
2021-02-18T02:36:37.4836178Z find translib -name \*\.go -exec sed -i -e 's/\"cvl/\"github.com\/Azure\/sonic-telemetry\/cvl/g' {} \;
2021-02-18T02:36:37.4837434Z sed -i -e 's/\.\.\/\.\.\/\.\.\/models\/yang/\.\.\/\.\.\/\.\.\/sonic-mgmt-framework\/models\/yang/' translib/ocbinds/oc.go
2021-02-18T02:36:37.4838804Z sed -i -e 's/\$GO run \$BUILD_GOPATH\/src\/github.com\/openconfig\/ygot\/generator\/generator.go/generator/' translib/ocbinds/oc.go
2021-02-18T02:36:37.4839673Z /usr/local/go/bin/go get github.com/openconfig/gnmi@89b2bf29312cda887da916d0f3a32c1624b7935f
2021-02-18T02:36:37.4840936Z go: downloading github.com/openconfig/gnmi v0.0.0-20190823184014-89b2bf29312c
2021-02-18T02:36:37.4841646Z /usr/local/go/bin/go get github.com/openconfig/ygot@724a6b18a9224343ef04fe49199dfb6020ce132a
2021-02-18T02:36:37.4842936Z go: downloading github.com/openconfig/ygot v0.6.1-0.20190723223108-724a6b18a922
2021-02-18T02:36:37.4843780Z /usr/local/go/bin/go get github.com/openconfig/goyang@064f9690516f4f72db189f4690b84622c13b7296
2021-02-18T02:36:37.4845172Z go: downloading github.com/openconfig/goyang v0.0.0-20190924211109-064f9690516f
2021-02-18T02:36:37.4846394Z go: downloading github.com/pborman/getopt v0.0.0-20190409184431-ee0cd42419d3
2021-02-18T02:36:37.4847106Z /usr/local/go/bin/go get github.com/openconfig/goyang@064f9690516f4f72db189f4690b84622c13b7296
2021-02-18T02:36:37.4847947Z /usr/local/go/bin/go get golang.org/x/crypto/ssh/terminal@e9b2fee46413
2021-02-18T02:36:37.4848984Z go: downloading golang.org/x/crypto v0.0.0-20191206172530-e9b2fee46413
2021-02-18T02:36:37.4850094Z go: found golang.org/x/crypto/ssh/terminal in golang.org/x/crypto v0.0.0-20191206172530-e9b2fee46413
2021-02-18T02:36:37.4851169Z go: downloading golang.org/x/sys v0.0.0-20190412213103-97732733099d
2021-02-18T02:36:37.4851837Z /usr/local/go/bin/go get github.com/jipanyang/gnxi@f0a90cca6fd0041625bcce561b71f849c9b65a8d
2021-02-18T02:36:37.4852894Z go: downloading github.com/jipanyang/gnxi v0.0.0-20181221084354-f0a90cca6fd0
2021-02-18T02:36:37.4853508Z /usr/local/go/bin/go install github.com/openconfig/ygot/generator
2021-02-18T02:36:37.4854553Z go: downloading github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
2021-02-18T02:36:37.4855144Z go: downloading google.golang.org/grpc v1.25.1
2021-02-18T02:36:37.4855686Z go: downloading github.com/golang/protobuf v1.3.2
2021-02-18T02:36:37.4856206Z go: downloading github.com/kylelemons/godebug v1.1.0
2021-02-18T02:36:37.4898299Z go: downloading golang.org/x/net v0.0.0-20191209160850-c0dbc17a3553
2021-02-18T02:36:37.4900475Z go: downloading google.golang.org/genproto v0.0.0-20190819201941-24fa4b261c55
2021-02-18T02:36:37.4901488Z go: downloading golang.org/x/text v0.3.0
2021-02-18T02:36:37.4903094Z /usr/local/go/bin/go get -x github.com/golang/glog@23def4e6c14b4da8ac2ed8007337bc5eb5007998
2021-02-18T02:36:37.4904314Z # get https://proxy.golang.org/github.com/golang/glog/@v/23def4e6c14b4da8ac2ed8007337bc5eb5007998.info
2021-02-18T02:36:37.4905646Z # get https://proxy.golang.org/github.com/golang/glog/@v/23def4e6c14b4da8ac2ed8007337bc5eb5007998.info: 200 OK (0.294s)
2021-02-18T02:36:37.4907056Z WORK=/tmp/go-build865181325
2021-02-18T02:36:37.4908348Z rm -rf vendor
2021-02-18T02:36:37.4909201Z /usr/local/go/bin/go mod vendor
2021-02-18T02:36:37.4910546Z go: downloading github.com/Workiva/go-datastructures v1.0.50
2021-02-18T02:36:37.4912261Z go: downloading github.com/go-redis/redis v6.15.6+incompatible
2021-02-18T02:36:37.4913176Z go: downloading github.com/antchfx/jsonquery v1.1.0
2021-02-18T02:36:37.4915068Z go: downloading github.com/c9s/goprocinfo v0.0.0-20191125144613-4acdd056c72d
2021-02-18T02:36:37.4916620Z go: downloading github.com/jipanyang/gnmi v0.0.0-20180820232453-cb4d464fa018
2021-02-18T02:36:37.4917570Z go: downloading github.com/antchfx/xmlquery v1.2.1
2021-02-18T02:36:37.4919361Z go: downloading github.com/google/gnxi v0.0.0-20191016182648-6697a080bc2d
2021-02-18T02:36:37.4920994Z go: downloading github.com/golang/groupcache v0.0.0-20191027212112-611e8accdfc9
2021-02-18T02:36:37.4922113Z go: downloading github.com/antchfx/xpath v1.1.2
2021-02-18T02:36:37.4923254Z ln -s vendor src
2021-02-18T02:36:37.4924729Z cp -r /tmp/go/pkg/mod/github.com/openconfig/gnmi@v0.0.0-20190823184014-89b2bf29312c/* vendor/github.com/openconfig/gnmi/
2021-02-18T02:36:37.4926645Z cp -r /tmp/go/pkg/mod/github.com/openconfig/goyang@v0.0.0-20190924211109-064f9690516f/* vendor/github.com/openconfig/goyang/
2021-02-18T02:36:37.4928587Z cp -r /tmp/go/pkg/mod/github.com/openconfig/ygot@v0.6.1-0.20190723223108-724a6b18a922/* vendor/github.com/openconfig/ygot/
2021-02-18T02:36:37.4931066Z cp -r /tmp/go/pkg/mod/golang.org/x/crypto@v0.0.0-20191206172530-e9b2fee46413 vendor/golang.org/x/crypto
2021-02-18T02:36:37.4932789Z chmod -R u+w vendor
2021-02-18T02:36:37.4933877Z patch -d vendor -p0 <patches/gnmi_cli.all.patch
2021-02-18T02:36:37.4934989Z patching file ./github.com/openconfig/gnmi/client/gnmi/client.go
2021-02-18T02:36:37.4936128Z patching file ./github.com/openconfig/gnmi/cmd/gnmi_cli/gnmi_cli.go
2021-02-18T02:36:37.4937457Z patching file ./github.com/openconfig/gnmi/client/query.go
2021-02-18T02:36:37.4938373Z patching file ./github.com/openconfig/gnmi/value/value.go
2021-02-18T02:36:37.4940209Z patch -d vendor/github.com/antchfx/jsonquery -p1 < ../sonic-mgmt-framework/patches/jsonquery.patch
2021-02-18T02:36:37.4941149Z patching file node.go
2021-02-18T02:36:37.4942927Z patch -d vendor/github.com/openconfig/goyang -p1 < ../sonic-mgmt-framework/goyang-modified-files/goyang.patch
2021-02-18T02:36:37.4943883Z patching file README.md
2021-02-18T02:36:37.4944492Z patching file annotate.go
2021-02-18T02:36:37.4945145Z patching file pkg/yang/entry.go
2021-02-18T02:36:37.4945757Z patching file yang.go
2021-02-18T02:36:37.4977967Z patch -d vendor/github.com/openconfig -p1 < ../sonic-mgmt-framework/ygot-modified-files/ygot.patch
2021-02-18T02:36:37.4979743Z patching file ygot/util/debug.go
2021-02-18T02:36:37.4980638Z patching file ygot/util/reflect.go
2021-02-18T02:36:37.4981391Z patching file ygot/util/schema.go
2021-02-18T02:36:37.4982115Z patching file ygot/ytypes/container.go
2021-02-18T02:36:37.4983042Z patching file ygot/ytypes/leaf.go
2021-02-18T02:36:37.4984089Z patching file ygot/ytypes/list.go
2021-02-18T02:36:37.4984860Z patching file ygot/ytypes/node.go
2021-02-18T02:36:37.4985637Z patching file ygot/ytypes/string_type.go
2021-02-18T02:36:37.4986814Z patching file ygot/ytypes/unmarshal.go
2021-02-18T02:36:37.4987827Z patching file ygot/ytypes/util_schema.go
2021-02-18T02:36:37.4988749Z patching file ygot/ytypes/validate.go
2021-02-18T02:36:37.4990969Z /usr/local/go/bin/go generate github.com/Azure/sonic-telemetry/translib/ocbinds
2021-02-18T02:36:37.4992762Z make -C /sonic/src/sonic-mgmt-framework/src/cvl/schema
2021-02-18T02:36:37.4994493Z make[3]: Entering directory '/sonic/src/sonic-mgmt-framework/src/cvl/schema'
2021-02-18T02:36:37.4995839Z Generating sonic-acl.yin ...
2021-02-18T02:36:37.4997304Z Generating sonic-interface.yin ...
2021-02-18T02:36:37.4998377Z Generating sonic-port.yin ...
2021-02-18T02:36:37.5000098Z Generating sonic-common.yin ...
2021-02-18T02:36:37.5001307Z Generating sonic-extension.yin ...
2021-02-18T02:36:37.5002917Z make[3]: Leaving directory '/sonic/src/sonic-mgmt-framework/src/cvl/schema'
2021-02-18T02:36:37.5004464Z make -C /sonic/src/sonic-mgmt-framework/models
2021-02-18T02:36:37.5006176Z make[3]: Entering directory '/sonic/src/sonic-mgmt-framework/models'
2021-02-18T02:36:37.5007837Z find: '../build/yaml': No such file or directory
2021-02-18T02:36:37.5009136Z mkdir -p ../build/rest_server/dist/.
2021-02-18T02:36:37.5011512Z cp -r ../tools/swagger_codegen/ui-dist ../build/rest_server/dist/ui
2021-02-18T02:36:37.5012940Z cp -r ../tools/swagger_codegen/go-server/src ../build/rest_server/dist/
2021-02-18T02:36:37.5013878Z touch ../build/rest_server/dist/.init_done
2021-02-18T02:36:37.5014697Z +++ Generating  landing page for Swagger UI +++
2021-02-18T02:36:37.5015473Z ../tools/ui_gen/src/uigen.py
2021-02-18T02:36:37.5077316Z make[3]: Leaving directory '/sonic/src/sonic-mgmt-framework/models'
2021-02-18T02:36:37.5078551Z make -C /sonic/src/sonic-mgmt-framework/models/yang
2021-02-18T02:36:37.5079759Z make[3]: Entering directory '/sonic/src/sonic-mgmt-framework/models/yang'
2021-02-18T02:36:37.5081115Z find: '../../models/yang/annotations/openapi_annotations': No such file or directory
2021-02-18T02:36:37.5081784Z +++++ Generating YAML files for Yang modules +++++
2021-02-18T02:36:37.5082768Z mkdir -p ../../build/yaml
2021-02-18T02:36:37.5083194Z pyang \
2021-02-18T02:36:37.5084038Z 	-f swaggerapi \
2021-02-18T02:36:37.5085133Z 	--outdir ../../build/yaml \
2021-02-18T02:36:37.5085970Z 	--plugindir ../../tools/pyang/pyang_plugins \
2021-02-18T02:36:37.5086870Z 	-p ../../models/yang/common:../../models/yang \
2021-02-18T02:36:37.5089500Z 	../../models/yang/openconfig-acl.yang ../../models/yang/openconfig-if-ethernet.yang ../../models/yang/openconfig-if-ip.yang ../../models/yang/openconfig-interfaces.yang ../../models/yang/openconfig-lldp.yang ../../models/yang/openconfig-platform.yang ../../models/yang/openconfig-system.yang 
2021-02-18T02:36:37.5092036Z ../../models/yang/openconfig-if-ethernet.yang:427: warning: node "openconfig-interfaces::state" is config false and is not part of the accessible tree
2021-02-18T02:36:37.5093208Z ('===> processing ', u'openconfig-acl')
2021-02-18T02:36:37.5094108Z ('===> processing ', u'openconfig-if-ethernet')
2021-02-18T02:36:37.5094978Z ('===> processing ', u'openconfig-if-ip')
2021-02-18T02:36:37.5095887Z ('===> processing ', u'openconfig-interfaces')
2021-02-18T02:36:37.5097051Z ('===> processing ', u'openconfig-lldp')
2021-02-18T02:36:37.5097979Z ('===> processing ', u'openconfig-platform')
2021-02-18T02:36:37.5098517Z Traceback (most recent call last):
2021-02-18T02:36:37.5099402Z   File "/usr/local/bin/pyang", line 533, in <module>
2021-02-18T02:36:37.5099874Z     run()
2021-02-18T02:36:37.5100324Z   File "/usr/local/bin/pyang", line 502, in run
2021-02-18T02:36:37.5100832Z     emit_obj.emit(ctx, modules, fd)
2021-02-18T02:36:37.5101396Z   File "../../tools/pyang/pyang_plugins/openapi.py", line 215, in emit
2021-02-18T02:36:37.5103228Z     with open(ctx.opts.outdir + '/' + module.i_modulename + ".yaml", "w") as spec:
2021-02-18T02:36:37.5105003Z IOError: [Errno 2] No such file or directory: u'../../build/yaml/openconfig-platform.yaml'
2021-02-18T02:36:37.5106443Z Makefile:62: recipe for target '../../build/yaml/.done' failed
2021-02-18T02:36:37.5107261Z make[3]: *** [../../build/yaml/.done] Error 1
2021-02-18T02:36:37.5108527Z make[3]: Leaving directory '/sonic/src/sonic-mgmt-framework/models/yang'
2021-02-18T02:36:37.5110215Z Makefile:26: recipe for target 'mgmt-deps' failed
2021-02-18T02:36:37.5111672Z make[2]: *** [mgmt-deps] Error 2
2021-02-18T02:36:37.5112760Z make[2]: Leaving directory '/sonic/src/sonic-telemetry'
2021-02-18T02:36:37.5113772Z dh_auto_build: make -j1 returned exit code 2
2021-02-18T02:36:37.5114878Z debian/rules:7: recipe for target 'build' failed
2021-02-18T02:36:37.5115440Z make[1]: *** [build] Error 2
2021-02-18T02:36:37.5116349Z make[1]: Leaving directory '/sonic/src/sonic-telemetry'
2021-02-18T02:36:37.5117380Z dpkg-buildpackage: error: debian/rules build gave error exit status 2
2021-02-18T02:36:37.5119046Z [  FAIL LOG END  ] [ target/debs/stretch/sonic-telemetry_0.1_amd64.deb ]
2021-02-18T02:36:38.2566638Z slave.mk:360: recipe for target 'target/debs/stretch/sonic-telemetry_0.1_amd64.deb' failed
2021-02-18T02:36:38.2576917Z make: *** [target/debs/stretch/sonic-telemetry_0.1_amd64.deb] Error 1
2021-02-18T02:36:38.2587831Z make: *** Waiting for unfinished jobs....
2021-02-18T02:38:39.0897124Z [ finished ] [ target/debs/stretch/libsnmp-base_5.7.3+dfsg-1.5_all.deb ] 
2021-02-18T02:49:43.0154099Z [ finished ] [ target/debs/stretch/sonic-mgmt-framework_1.0-01_amd64.deb ] 
2021-02-18T03:55:37.5808659Z [ finished ] [ target/debs/stretch/linux-headers-4.9.0-14-2-common_4.9.246-2_all.deb ] 
2021-02-18T03:55:42.3577208Z make[1]: *** [Makefile.work:202: target/sonic-broadcom.bin] Error 2
2021-02-18T03:55:42.3578131Z make[1]: Leaving directory '/agent/_work/1/s'
2021-02-18T03:55:42.3579156Z make: *** [Makefile:8: target/sonic-broadcom.bin] Error 2
2021-02-18T03:55:42.4132638Z ##[error]Bash exited with code '2'.
2021-02-18T03:55:42.4157289Z ##[section]Finishing: Build sonic image
```

#### Steps to reproduce the issue:
1.
2.
3.

#### Output of `show version`:

```
(paste your output here)
```

**Comment by seiferteric** [comment]
This PR (https://github.com/Azure/sonic-buildimage/pull/6910) to add mgmt-framework as a build dependency should fix the issue.

**Comment by qiluo-msft** [comment]
Thanks!
